app/uk/gov/hmrc/exports/controllers/SubmissionController.scala

```diff
@@ -41,7 +42,25 @@ class SubmissionController @Inject()(
       authorizedWithEnrolment[SubmissionResponse](_ => processRequest)
     }
 
-  def processRequest()(implicit request: Request[SubmissionResponse], hc: HeaderCarrier): Future[Result] = {
+  def saveMovementSubmission(): Action[MovementResponse] =
+    Action.async(parse.json[MovementResponse](MovementResponse.format)) { implicit request =>
```

Comment:
My propose is making format as implicit instead of passing it here. In companion object `MovementResponse`